src/w3c/templates/headers.js

```diff
@@ -2,16 +2,42 @@ import "deps/hyperhtml";
 import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
+import { pub } from "core/pubsubhub";
 
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem =
+    document.querySelector("h1#title") || document.createElement("h1");
```

Comment:
Generally okay, but maybe we can split these into `getSpecTitleElement()` and `getSpecSubtitleElement()` functions for readability.

Reply:
If I could get a confirmation on this it, I'll incorporate :sweat_smile:

Reply:
Good idea about splitting this out into functions.